schema/method_units/mathematical/opt/diff/ordern.json

```diff
@@ -0,0 +1,20 @@
+{
+    "schemaId": "method-unit-opt-diff-ordern",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "description": "Mixed order and higher order algorithms for the optimization of differentiable functions",
```

Comment:
I had to create a new tier for mixed or higher order optimization methods as conjugate gradient is often described as order 1.5. See here for instance: https://optimization.cbe.cornell.edu/index.php?title=Conjugate_gradient_methods